packages/cli/lib/projects.js

```diff
@@ -243,7 +249,7 @@ const pollBuildStatus = async (accountId, name, buildId) => {
         });
       }
 
-      if (isBuildComplete(buildStatus)) {
+      if (isBuildComplete(buildStatus) && spinnies.hasActiveSpinners()) {
```

Comment:
Why do we care if there are active spinners here?

Reply:
Oof, I think this is left over from testing something else out 👍